features/cart/viewing_cart/viewing_cart_summary.feature

```diff
@@ -0,0 +1,31 @@
+@viewing_cart
+Feature: Viewing a cart summary
+    In order to see details about my order
+    As a visitor
+    I want to be able to see my cart summary
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @ui
+    Scenario: Viewing information about empty cart
+        When I open cart summary page
+        Then I should be notified that my cart is empty
```

Comment:
These steps were for verifying that there is a notification about success or validation error. He it is just fine to say that cart is empty, which does not necessarily mean that there is a "notification" about. Nothing has happened, you just view empty cart. These scenarios can also be executed against API, and there is no notification at all, just empty cart.